pkg/external_repos/introspect_test.go

```diff
@@ -174,10 +181,11 @@ func TestUpdateIntrospectionStatusMetadata(t *testing.T) {
 				LastIntrospectionUpdateTime:  &timestamp,
 				LastIntrospectionError:       pointy.String(""),
 				Status:                       config.StatusValid,
+				PackageCount:                 *pointy.Int(100),
```

Comment:
I haven't tested it, but I'm pretty sure `*pointy.Int(100)` can just be `100`

There are a few places where this `*pointy.Int(100)` exists, but I'll just leave the comment here.

Reply:
you are right!! abusing the IDE features! :+1:  I have removed all of them.